docs/hydra/guides/logout.mdx

```diff
@@ -4,26 +4,32 @@ title: Implementing the OIDC logout endpoint & UI
 sidebar_label: OpenID Connect user logout
 ---
 
-When using Ory OAuth2 and OpenID Connect, the default logout experience is the managed account experience. However, it's possible
-to customize the logout experience by implementing a custom logout endpoint using the Ory SDK. In this guide, we will walk through
-the steps of implementing your own logout endpoint and customizing the logout UI.
+When using Ory OAuth2 and OpenID Connect, by default the logout operation is handled by the Ory Account Experience. Alternatively,
+you can customize the logout experience by implementing a custom logout endpoint using the Ory SDK.
+
+This document shows how to implement your own logout endpoint and customize the logout UI.
 
 ## OpenID Connect logout overview
 
-OpenID Connect defines two types of logout mechanisms: backchannel logout and frontchannel logout. Backchannel logout is initiated
-by the identity provider (IdP) and is a server-to-server call. Frontchannel logout, on the other hand, is initiated by the client
-and is a client-to-server call.
+OpenID Connect defines two types of logout mechanisms: back-channel logout and front-channel logout.
+
+- Back-channel logout is initiated by the identity provider (IdP) and is a server-to-server call.
+- Front-channel logout, is initiated by the client and is a client-to-server call.
 
 ## Customize logout experience
 
-To customize the logout experience, we need to implement a custom logout endpoint that can handle backchannel and frontchannel
-logout requests. The Logout HTML Form can't be a Single Page App (Client-side browser application) or a Mobile App! There has to
-be a server-side component with access to an Ory API Key.
+To customize the logout experience, you must implement a custom logout endpoint that can handle back-channel and front-channel
+logout requests. The logout HTML Form can't be a Single Page App (Client-side browser application) or a mobile. There has to be a
```

Comment:
Not sure why you dropped "app" from "mobile app" here? It's not clear to me what "a mobile" is.

Reply:
That's a mistake, must've removed it by accident when I was updating capitalization.

Reply:
Fixed by applying Vince's comment.